### PR TITLE
Update validationDefault

### PR DESCRIPTION
**Pull request by yosus**
Updated  validationDefault from protected to public

### CODE REVIEW

File: en/core-libraries/form.rst [hunk 1]
```diff
@@ -130,6 +130,9 @@ Values set with this method will overwrite existing data in the form object::
                     'name' => 'John Doe',
                     'email' => 'john.doe@example.com'
                 ]);
+                // Get the values like this
+                $allValues = $contact->getData();
```
Comment (othercorey) [inline en/core-libraries/form.rst]: I don't know if this example makes sense to put here. It's not related to the rest of the code.

Comment (markstory) [reply en/core-libraries/form.rst]: It could be in a separate example?

Comment (othercorey) [reply en/core-libraries/form.rst]: I would make them separate if it's important to include.

Comment (yosus) [reply en/core-libraries/form.rst]: ok, sure. let me create a separate getData sample.

**Review by yosus — COMMENTED** [review]
Updated minLength in validator
Added getting-data seperate sample
I would like to note that in cake 3.9.6, calling `execute` does not automatically save the data. After cake 4.1.0, calling `execute` will automatically save the data.

Current doc flow is :

1. Create Form
2. **Processing Data**
3. Setting Data
4. Getting Data
5. Getting Errors


Would it be better to put Processing-Data nearer Getting-Errors. Like this :

1. Create Form
2. Setting Data
3. Getting Data
4. **Processing Data**
5. Getting Errors

**Comment by markstory** [comment]
> Would it be better to put Processing-Data nearer Getting-Errors.

That sounds like a better structure to me :+1:

**Review by yosus — COMMENTED** [review]
Relocated `Processing Request Data` section

**Comment by markstory** [comment]
Thank you 👍

**Pull request merged**